src/frontend/komponenter/Fagsak/Personlinje/Behandlingsmeny/OpprettBehandling/useOpprettBehandling.ts

```diff
@@ -330,19 +327,16 @@ const useOpprettBehandling = (
         nullstillSkjema();
     };
 
-    const dagenførForrigeMånedStartet = () => {
-        const dato = new Date();
-        dato.setMonth(dato.getMonth() - 1);
-        dato.setDate(0);
-        return dato;
+    const maksdatoForMigrering = () => {
```

Comment:
Dette kunne ha vært en `const MAKSDATO_FOR_MIGRERING =new Date('2023-01-01');`

Både fordi all caps indikerer en statisk/konstant verdi og fordi vi slipper å opprette + kalle på en funksjon :)

Reply:
evt `const MAKSTIDSPUNKT_FOR_MIGRERING = Date.parse('2023-01-01');`, så slipper du å kjøre `.getTime()` i `erDatoMindreEllerLikMaksdato`

Reply:
Med mindre denne verdien brukes utenfor denne filen, da er kanskje Date-objektet riktigst? Du får se hva som føles best

Reply:
Brukes utenfor, så gikk for Date-objekt konstant 👍